server/database/queries/updateVoterStatus.js

```diff
@@ -1,15 +1,8 @@
 const dbConnection = require('../dbConnection');
-
-const updateVoterStatus = (idOfVoter) => {
-  const queryString = 'UPDATE voters SET status = 3 WHERE id = $1';
-  dbConnection.query(queryString, [idOfVoter], (err) => {
-    if (err) {
-      console.log(err);
-    } else {
-      // this is just for check - remove else before push
-      console.log('success');
-    }
-  });
+// here it set the status as what is sent to her from the frontend
```

Comment:
Maybe this comment could be formulated better, I'm not sure I understand what you mean... What is the status and what are the possible values?